app/services/events/stores/clickhouse_store.rb

```diff
@@ -566,6 +566,9 @@ def weighted_sum_breakdown(initial_value: 0)
               ]
             )
           ).rows
+          rows.map do |row|
+            [row[0], row[1], row[2], row[3].to_i, row[4]]
+          end
```

Comment:
could we avoid `map` returning the expected values from the SQL query? we could use clickhouse cast functions to achieve the `to_i` call.